homeassistant/components/vulcan/calendar.py

```diff
@@ -107,8 +108,12 @@ async def async_get_events(
         event_list = []
         for item in events:
             event = CalendarEvent(
-                start=datetime.combine(item["date"], item["time"].from_),
-                end=datetime.combine(item["date"], item["time"].to),
+                start=datetime.combine(item["date"], item["time"].from_).astimezone(
+                    now().tzinfo
+                ),
```

Comment:
If you want set the user configured timezone on this, you can import `DEFAULT_TIME_ZONE` from `homeassistant.util.dt` instead.

```suggestion
                start=datetime.combine(item["date"], item["time"].from_).astimezone(
                    DEFAULT_TIME_ZONE
                ),
```

That said, is the timezone of the upstream vulcan cloud the same as of the timezone the user configured in Home Assistant? Sounds a bit odd...?

../Frenck

Reply:
Yeah, you are right, i've changed the code to always use "Europe/Warsaw" datetime as that's where the company is located.